### ISSUE TITLE
Hadolint always exits with 0 even when there are parsing errors

### ISSUE THREAD

**Issue by w131996** [hadolint/hadolint#938]
- [x] This is a bug report
- [ ] This is a feature request
- [ ] I searched existing issues before opening this one

In my test, I added an incorrect instruction for instance `AAA` in the Dockerfile and ran a command to lint the Dockerfile with hadolint. Then, I printed the exit code with $? and got an 0 exit code.

```bash
# hadolint /path/to/Dockerfile
/path/to/Dockerfile:6:1 unexpected 'A' expecting '#', '\', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, a pragma, at least one space, or end of input
# $?
sh: 0: command not found
```

I did the same test but with `RUNNN` and I got the following result:
```bash
# hadolint /path/to/Dockerfile
/path/to/Dockerfile:6:1 missing whitespace
# $?
sh: 0: command not found
```

Should hadolint return a non-zero exit code in these cases ?

**Comment by w131996** [comment]
Configuration issue on my side.
